src/main/java/com/alkemy/ong/data/gateways/DefaultNewsGateway.java

```diff
@@ -0,0 +1,38 @@
+package com.alkemy.ong.data.gateways;
+
+import com.alkemy.ong.data.entities.NewsEntity;
+import com.alkemy.ong.data.repositories.NewsRepository;
+import com.alkemy.ong.domain.news.NewsGateway;
+import com.alkemy.ong.domain.news.News;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DefaultNewsGateway implements NewsGateway {
+
+    private final NewsRepository newsRepository;
+
+    public DefaultNewsGateway(NewsRepository newsRepository) {
+        this.newsRepository = newsRepository;
+    }
+
+    @Override
+    public News findById(Long newsId) {
+
+        NewsEntity newsEntity = newsRepository.findById(newsId).orElse(null);
```

Comment:
la idea es q cuando el id q buscamos no existe, tiremos una excepción y q en el cliente reciba un 404. Este código es el q se usa para Resource Not Found.
Te animas a implementar un global exception handler q se encargue de centralizar todo el uso de excepciones para devolver los errores correctos de HTTP?
https://www.baeldung.com/exception-handling-for-rest-with-spring
fíjate la opción 4. Solution 3: @ControllerAdvice
La idea es q crees una CustomException q se llame ResourceNotFoundException. Ponela en domain bajo un paquete exceptions.
Dsp en la capa web, crea la clase GlobalExceptionHandler como dice la pagina, y vas a mapear esa excepción para q retorne un 404. 
Entonces acá, en lugar de usar orElse(Null) vas a usar orElseThrow(ResourcNotFoundException) con el mensaje de q el id no existe.
Y asi vamos a manejar excepciones d forma centralizada!
Para probarlo, cuando pongas un id q no existe, en postman deberías recibir el status code 404